classes/request.class.php

```diff
@@ -193,6 +196,7 @@ public function send_request($endpoint, $requestbody, $method, $requesttype = 'g
             if (json_last_error() !== JSON_ERROR_NONE) {
                 $result = new stdClass();
                 $httpstatus = 400;
+                $this->logger->error('The JSON returned was not valid.');
```

Comment:
Should we include the JSON in the log so we can see the invalid JSON?

Reply:
I'll look into this one. It might be a good idea. In the code I see we already log the JSON but if it's invalid we create a new JSON object with just an httpstatus. Would be nice to log the invalid JSON I agree.